PiwikTracker/PiwikTracker.swift

```diff
@@ -22,6 +22,8 @@ final public class PiwikTracker: NSObject {
     private var queue: Queue
     internal let siteId: String
     
+    public var logger: Logger? = DefaultLogger(minLevel: .warning)
+    
     internal static var _sharedInstance: PiwikTracker?
```

Comment:
I would not use '_' in any variable or instance names.

Often if you find yourself facing problems with naming resorting to using `_` or other type of prefix/post fix to avoid naming conflicts it is an indication of a design problem.